### DISCUSSION TITLE
Change formatting of Cite this Page in menu

### DISCUSSION THREAD

**Discussion by rrolonNelson** [thegetty/quire#875] [Get Help]
Howdy, 
I was curious if there was more documentation on the cite this page section of the side menu?  It looks like the way that the Chicago and MLA formats are done are different than what our team would like.  I have looked at the different citation packages https://github.com/lewisacidic/citation-styles/tree/master/packages (the project already uses the mla and Chicago style which is what we want but need a slightly different style).  I have also seen that there are additional csl styles https://github.com/citation-style-language/styles but I am having trouble figuring out how to add them to the project to get the right formatting. 

This is what the team is expecting as our standard: 
Chicago: Author First Name Last Name, “Page title,” biography, in Aimee Marcereau DeGalan, Blythe Sobol, and Maggie Keenan, The Starr Collection of Portrait Miniatures, 1500–1850: The Collections of the Nelson-Atkins Museum of Art, ed. Aimee Marcereau DeGalan (Kansas City, MO: Nelson-Atkins Museum of Art, 2023), https://doi.org/10.37764/8322.5000. 

MLA: Author Last Name, First Name. “Page title,” biography. Aimee Marcereau DeGalan, Blythe Sobol, and Maggie Keenan. The Starr Collection of Portrait Miniatures, 1500–1850: The Collections of the Nelson-Atkins Museum of Art, edited by Aimee Marcereau DeGalan, Nelson-Atkins Museum of Art, 2023. doi: https://doi.org/10.37764/8322.5000. 

This is what we currently get on the cite this page section:
Chicago
Zugazagoitia, Julián. “Director’s Foreword.” In The Starr Collection of Portrait Miniatures, 1500–1850: The Collections of The Nelson-Atkins Museum of Art (Volume I: Continental and American), by Aimee Marcereau DeGalan. Kansas City: The Nelson-Atkins Museum of Art, 2023. https://wwwstag.nelson-atkins.org/starr1/foreword/.
MLA
Zugazagoitia, Julián. “Director’s Foreword.” The Starr Collection of Portrait Miniatures, 1500–1850: The Collections of The Nelson-Atkins Museum of Art (Volume I: Continental and American), by Aimee Marcereau DeGalan, The Nelson-Atkins Museum of Art, 2023, https://wwwstag.nelson-atkins.org/starr1/foreword/. Accessed 1 Nov. 2023.

**Comment by Erin-Cecele** [comment]
Hi @rrolonNelson, Quire uses a piece of software called [simple cite](https://github.com/lewisacidic/simple-cite#readme) to format the citations. Quire just feeds it the data, and it returns it in what should be the right format. You can find more information here: https://github.com/thegetty/quire/blob/main/packages/11ty/_plugins/citations/README.md. 

Things like the capitalization of the title and maybe also the placement of the author name are controlled through these files in Quire projects, which we added into Quire unchanged from an external source as part of our implementation of simple cite:
 
`plugins/citations/styles/chicago-fullnote-bibliography.js`
`plugins/citations/styles/mla.js`
 
It seems possible that these files could be manually edited to output the citations as you like them, or perhaps there’s another [variant citation format](https://github.com/citation-style-language/styles) that could be used, but that’s well beyond our scope. Another potential option could be to code in the structured citations directly and circumvent the use of simple cite. You'd just be adding it to the templates something like this.
 
first_name last_name, "page.title." In publication.title. publication.subtitle ....

I hope that's helpful!
